torch_complex/functional.py

```diff
@@ -46,6 +46,9 @@ def einsum(equation, operands):
     >>> numpy.testing.assert_allclose(test.numpy(), valid)
 
     """
+    if len(operands) == 1 and isinstance(operands, (tuple, list)):
```

Comment:
```suggestion
    if len(operands) == 1:
```

`operands` is always tuple if * is added.